lib/polymer_datepicker.html

```diff
@@ -139,7 +139,7 @@
           <paper-input disabled="[[disabled]]" id="input" on-click="showPicker" required="[[required]]" auto-validate="" value="{{textDate}}" label="[[label]]" on-blur="inputLeft" class$="[[computeInputClass(dateonly)]]"></paper-input>
         </div>
         <div style="position:relative">
-            <date-picker-overlay id="pick" style="position:absolute;left:0;top:0" autofocusdisabled="true" autoclosedisabled="true" opened="[[pickerOpen]]" on-iron-overlay-opened="doPos" on-mousedown="cancelClose">
+            <date-picker-overlay id="pick" style="position:absolute;left:0;top:0" autofocusdisabled="true" autoclosedisabled="true" opened="[[pickerOpen]]" on-iron-overlay-opened="doPos" on-iron-overlay-closed="doClose" on-mousedown="cancelClose">
```

Comment:
Tnx. Probably it would suffice to change the binding to `pickerOpen`. Can you try with `{{pickerOpen}}` instead of `[[pickerOpen]]` ?